src/core_ocean/mode_init/mpas_ocn_init_global_ocean.F

```diff
@@ -996,8 +998,8 @@ subroutine ocn_init_setup_global_ocean_create_model_topo(domain, iErr)!{{{
                 end do
 
                 if (maxLevel < maxLevelCell(iCell) ) then
-                   smoothedLevelsField % array(iCell) = maxLevel + 1
-                   bottomDepth(iCell) = refBottomDepth(maxLevel + 1)
+                   smoothedLevelsField % array(iCell) = maxLevel
```

Comment:
@mark-petersen I tested this code and there are still ~700 locations that need adjustment (pits).  In my offline code I had to iterate twice to remove all issues.  Can you add a loop around this adjustment?

Reply:
@mark-petersen, the loop needs to be until there are no more pits to fill (with a maximum of `maxEdges` because I believe that's the max).